boot/menhir/menhirLib.mli

```diff
@@ -1793,5 +1793,5 @@ module MakeEngineTable
      and type nonterminal = int
 end
 module StaticVersion : sig
-val require_20240715: unit
+val require_20231231: unit
```

Comment:
You are downgrading the menhir version, this should not happen. Can you regenerate your patch with a more up-to-date menhir?